metaseq/model_parallel/modules/sequence_parallel_transformer_layer.py

```diff
@@ -0,0 +1,299 @@
+# Copyright (c) Meta Platforms, Inc. and affiliates. All Rights Reserved.
+#
+# This source code is licensed under the MIT license found in the
+# LICENSE file in the root directory of this source tree.
+
+import math
+import torch
+
+import fused_layer_norm_cuda
```

Comment:
Is this coming from apex installation?  Test are failing to import this...

Reply:
hm, why isn't it megatron.fused_kernels.fused_layer_norm?

Reply:
sorry its coming from apex, https://github.com/NVIDIA/apex/blob/master/apex/normalization/fused_layer_norm.py#L36-L37